symbols/overlay29.yml

```diff
@@ -306,6 +306,26 @@ overlay29:
         
         r0: pointer to entity
         returns: pointer to tile
+    - name: UpdateEnityPixelPos
+      address:
+        EU: 0x22E2380
+        NA: 0x22E1A40
+      description: |-
+        Updates an entity's pixel_pos field using the specified pixel_position struct, or its own pos field if it's null.
+        
+        r0: Entity pointer
+        r1: Pixel position to use, or null to use the entity's own position
+    - name: InitEnemyEntity
```

Comment:
The convention with `Init*` functions is that they take a pointer to initialize in-place, and usually have no return value. If this actually spawns the enemy on the floor somewhere, I think `SpawnEnemyEntity` would make more sense. If not, maybe we could rename this to something like `CreateEnemyEntity`?

Reply:
I didn't go with `SpawnEnemyEntity` because it doesn't spawn the enemy, that's handled by `SpawnMonster`, which calls this function. `CreateEnemyEntity` sounds good, I'll change it.